lms/djangoapps/edraak_university/forms.py

```diff
@@ -0,0 +1,44 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from models import UniversityID
+
+
+class UniversityIDForm(forms.ModelForm):
+    full_name = forms.CharField(
+        label=_('Full Name *'),
+        help_text=_(
+            'Enter your full name that you use at the university (same as your profile page).'
+        ),
+        required=True,
+        min_length=2,
+    )
+
+    university_id = forms.CharField(
+        label=_('University ID *'),
+        help_text=_('Enter the full university ID e.g. 201311318.'),
```

Comment:
university ID -> student university ID
As a suggestion: FIXME: Make ID format instruction course-variant, so coordinators can define it for each course.